src/autopas/autopasIncludes.h

```diff
@@ -20,7 +20,7 @@
 #include "autopas/utils/WrapOpenMP.h"
 
 // particles
-#include "autopas/particles/MoleculeLJ.h"
+#include "autopas/molecularDynamics/MoleculeLJ.h"
```

Comment:
remove this here and move to separate `autopasmd.h` or so. similar to the sph file.

Reply:
nope at least not in this PR